### PR TITLE
Feedback: backend implementation

### PR DESCRIPTION
**Pull request by markprudnikov**

### CODE REVIEW

File: app-backend/src/main/kotlin/org/example/appbackend/dto/FeedbackDto.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+data class FeedbackDto(
+    val name: String?,
+    val email: String?,
+    val phone: String?,
+    val companyName: String?,
+    val rating: Int?,
+    val comment: String?
+)
+
+fun FeedbackEntity.toDto() = FeedbackDto(name, email, phone, companyName, rating, comment)
```
Comment (e2e4b6b7) [inline app-backend/src/main/kotlin/org/example/appbackend/dto/FeedbackDto.kt:14]: this staff is autogenerated using mapper (mappers packege) (slightly better design + unification with other entity-dto mapping)

**Review by e2e4b6b7 — CHANGES_REQUESTED** [review]
You should also write changelog for db (see src/main/resources) (also take index 4 as 3rd will be acquired by my PR)

File: app-backend/src/main/kotlin/org/example/appbackend/dto/FeedbackDto.kt (hunk 1 above)
Comment (markprudnikov) [reply app-backend/src/main/kotlin/org/example/appbackend/dto/FeedbackDto.kt:14]: thank you!

File: app-backend/src/main/resources/db/changelog/4_add_feedback.yml.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: your_name
```
[the remark this answers is not in the document]
Comment (markprudnikov) [reply app-backend/src/main/resources/db/changelog/4_add_feedback.yml.yaml:4]: hahahhahha

[the remark this answers is not in the document]
Comment (markprudnikov) [reply app-backend/src/main/resources/db/changelog/4_add_feedback.yml.yaml:4]: hahahhahha

File: app-backend/src/main/resources/db/changelog/4_add_feedback.yml.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
```
[the remark this answers is not in the document]
Comment (markprudnikov) [reply app-backend/src/main/resources/db/changelog/4_add_feedback.yml.yaml:3]: true

[the remark this answers is not in the document]
Comment (markprudnikov) [reply app-backend/src/main/resources/db/changelog/4_add_feedback.yml.yaml:3]: thanks

**Pull request merged**